resources/scripts/make-wallets.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Function to print a message in a box
+print_in_box() {
+    local message="$1"
+    local border_length=$((${#message} + 4))
+    local border=$(printf '%*s' "$border_length" | tr ' ' '#')
+
+    echo "$border"
+    echo "# $message #"
+    echo "$border"
+}
+
+# Check for correct number of arguments
+if [ "$#" -ne 2 ]; then
+    echo "Usage: $0 <number_of_wallets> <amount_per_wallet>"
```

Comment:
if it requires arguments to be passed in,
better give an example of usage in the comment (head of this file, or in the readme correspondently?)